core/meta/inc/TClass.h

```diff
@@ -309,12 +311,14 @@ friend class TProtoClass;
    TClass(const TClass& tc) = delete;
    TClass& operator=(const TClass&) = delete;
 
+   void LoadClassInfo() const;
```

Comment:
You are right, I looked up in the file to check for public/private, but hit a `public:` label inside `InsertTClassInRegistryRAII` class that fooled me into thinking LoadClassInfo() was not private.
Good catch. I will drop this commit.